README.md

```diff
@@ -29,6 +29,13 @@ Create a .env file
 cp .env.example .env # Fill out all the necessary fields
 ```
 
+## 🔑 Getting your API key
+Head to the [API keys](https://creator.tebex.io/developers/api-keys) page within your creator panel. You'll find a private key and a public token.
+
+Inside your **.env`** file, do the following:
```

Comment:
Please can you remove the ` after .env? Thanks!

Reply:
Fixed! Thanks for spotting